src/mrack/transformers/transformer.py

```diff
@@ -41,9 +42,13 @@ async def init(self, cfg, metadata):
         self._provider = providers.get(self._config_key)
         await self.init_provider()
 
+    def set_display_name(self, name):
```

Comment:
It's a public property, I think this is not necessary,

Reply:
allright